source/D0323r5.bs

```diff
@@ -108,51 +113,36 @@ constexpr explicit unexpected(E &&);
     constexpr E const&& value() const&&;
 ```
 
-*Returns*: `move(val)`.
+*Returns*: `std::move(val)`.
 
 
 
-�.�.4 Unexpected relational operators [*unexpected.relational_op*] {#unexpected.relational_op}
+�.�.4 Unexpected equality operators [*unexpected.relational_op*] {#unexpected.relational_op}
 ------------------------------------------------------------------
 
 ```c++
     template <class E>
         constexpr bool operator==(const unexpected<E>& x, const unexpected<E>& y);
 ```
 
-
-*Requires*: `E` shall meet the requirements of *EqualityComparable*.
-
-*Returns*: `x.value() == y.value()`.
-
-*Remarks*: Specializations of this function template, for which `x.value() ==
-y.value()` is a core constant expression, shall be constexpr functions.
+*Effects*: Equivalent to `return x.value() == y.value()`.
 
 ```c++
     template <class E>
         constexpr bool operator!=(const unexpected<E>& x, const unexpected<E>& y);
 ```
-*Requires*: `E` shall meet the requirements of *EqualityComparable*.
-
-*Returns*: `x.value() != y.value()`.
-
-*Remarks*: Specializations of this function template, for which `x.value() !=
-y.value()` is a core constant expression, shall be constexpr functions.
+*Effects*: Equivalent to `return x.value() != y.value()`.
 
 �.� Expected objects [expected] {#expected}
 -------------------------------
 
 �.�.1 In general [*expected.general*] {#expected.general}
 -------------------------------------
 
-This subclause describes class template expected that represents expected
-objects. An `expected<T, E>` object is an object that contains the storage for
-another object and manages the lifetime of this contained object `T`,
-alternatively it could contain the storage for another unexpected object
-`E`. The contained object may not be initialized after the expected object has
-been initialized, and may not be destroyed before the expected object has been
-destroyed. The initialization state of the contained object is tracked by the
-expected object.
+This subclause describes class template `expected` that represents expected
+objects. An `expected<T, E>` object holds an object of type `T` or 
+an object of type `unexpected<E>` and manages the lifetime of the contained
```

Comment:
Here do we want to say `unexpected<E>`, or use `E`? It's a bit weird since `unexpected<E>` already contains an `E`.

Reply:
We need to say `unexpected<E>` if we want to be able to have `expected<int,int>`.